server/plugin/virtual_agent.go

```diff
@@ -219,15 +251,45 @@ func (p *Plugin) ProcessResponse(data []byte) error {
 					return err
 				}
 			}
-		//TODO: Modify later to display a proper card.
 		case *OutputCard:
-			var data OutputCardData
-			if err = json.Unmarshal([]byte(res.Data), &data); err != nil {
-				return err
-			}
+			switch res.TemplateName {
+			case OutputCardSmallImageType, OutputCardLargeImageType:
+				var data OutputCardImageData
+				if err = json.Unmarshal([]byte(res.Data), &data); err != nil {
+					return err
+				}
 
-			if _, err = p.DMWithAttachments(userID, p.CreateOutputCardAttachment(&data)); err != nil {
-				return err
+				if _, err = p.DMWithAttachments(userID, p.CreateOutputCardImageAttachment(&data)); err != nil {
+					return err
+				}
+			case OutputCardVideoType:
+				var data OutputCardVideoData
+				if err = json.Unmarshal([]byte(res.Data), &data); err != nil {
+					return err
+				}
+
+				if _, err = p.DMWithAttachments(userID, p.CreateOutputCardVideoAttachment(&data)); err != nil {
+					return err
+				}
+
+				channel, err := p.API.GetDirectChannel(userID, p.botUserID)
+				if err != nil {
+					p.API.LogInfo("Couldn't get bot's DM channel with user", "userID", user)
```

Comment:
for this log i think we should not print the whole user just the userID or username is enough.